src/newSeed/baseStage.ts

```diff
@@ -43,13 +44,16 @@ export abstract class BaseStage {
   }
 
   activate(_params: unknown, routeConfig: RouteConfig): void {
+    this.nextButtonClicked = false;
     Object.assign(this, routeConfig.settings);
   }
 
   async detached(): Promise<void> {
-    const message = await this.validateInputs();
-    if (!message) {
-      this.persistData();
+    if (!this.nextButtonClicked) {
```

Comment:
Only call the validate input function if the next button has not been clicked, this means that the navigation away was from the side-bar